README.md

```diff
@@ -1,333 +1,83 @@
 
+Permission sets are stored in AWS SSO and are only used for AWS accounts. They are not used to manage access to cloud applications. Permission sets ultimately get created as IAM roles in a given AWS account, with trust policies that allow users to assume the role through AWS SSO.
 
 ## Usage
 
-
-
-This module contains two sub-modules that can be used in conjunction to provision AWS SSO Permission Sets and to 
-assign AWS SSO Users and Groups to Permissions Sets in accounts.
-
-- [modules/account-assignments](/modules/account-assignments) - a module for assigning users and groups to permission 
-sets in particular accounts
-- [modules/permission-sets](/modules/permission-sets) - a module for provisioning AWS SSO permission sets
-
-
-
-
-## Examples
-
-Here is a full example of using these modules to provision permission sets and assign them to accounts:
-- [`examples/complete`](/examples/complete) - complete example of using these modules
-
-
-
-<!-- markdownlint-disable -->
-## Makefile Targets
-```text
-Available targets:
-
-  help                                Help screen
-  help/all                            Display help for all targets
-  help/short                          This help short screen
-  lint                                Lint terraform code
+```hcl
+module "permission_sets" {
+  source = "https://github.com/nimbux911/terraform-aws-sso.git?ref=main"
+
+  permission_sets = [
+    {
+      name               = "SSO_PS_ADMINISTRATOR",
+      description        = "Allow Full Access to the account",
+      relay_state        = "",
+      session_duration   = "PT12H",
+      tags               = {},
+      inline_policy      = "",
+      policy_attachments = ["arn:aws:iam::aws:policy/AdministratorAccess"]
+      customer_managed_policy_attachments = [{
+        name = aws_iam_policy.S3Access.name
+        path = aws_iam_policy.S3Access.path
+      }]
+    },
+    {
+      name                                = "S3AdministratorAccess",
+      description                         = "Allow Full S3 Admininstrator access to the account",
+      relay_state                         = "",
+      session_duration                    = "",
+      tags                                = {},
+      inline_policy                       = data.aws_iam_policy_document.S3Access.json,
+      policy_attachments                  = []
+      customer_managed_policy_attachments = []
+    }
+  ]
+}
+
+data "aws_iam_policy_document" "S3Access" {
+  statement {
+    sid = "1"
+
+    actions = ["*"]
+
+    resources = [
+      "arn:aws:s3:::*",
+    ]
+  }
+}
+
+resource "aws_iam_policy" "S3Access" {
+  name   = "S3Access"
+  path   = "/"
+  policy = data.aws_iam_policy_document.S3Access.json
+  tags   = module.this.tags
+}
+
+account_assignments = [
```

Comment:
Shouldn't this assignation be inside the module instantiation? (It's a module input)